lib/airrecord/query_string.rb

```diff
@@ -20,8 +20,6 @@ def self.escape(*query)
     end
 
     module Encodings
-      using QueryString
```

Comment:
This line is left over from an earlier draft that used Refinements. Turns out trying to use Refinements from a module that doesn't declare any works fine under CRuby, but caused #47 under JRuby.